example/lib/main.dart

```diff
@@ -69,6 +71,46 @@ class _MyAppState extends State<MyApp> {
     CleverPush.shared.showTopicsDialog();
   }
 
+  void _getNotifications() async {
+    var notifications = await CleverPush.shared.getNotifications();
+    if (notifications.isNotEmpty) {
+      print(notifications[0]);
+    }
+    this.setState(() {
+      _debugLabelString = notifications.length.toString();
+    });
+  }
+
+  void _getSubscriptionTopics() async {
+    var topicIds = await CleverPush.shared.getSubscriptionTopics();
+    String topicIdsString = "";
+    for (var i = 0; i < topicIds.length; i++) {
+      if (topicIdsString.isEmpty) {
+        topicIdsString = topicIds[i];
+      } else {
+        topicIdsString = topicIdsString + "," + topicIds[i];
+      }
+    }
+    this.setState(() {
+      _debugLabelString = topicIdsString;
+    });
+  }
+
+  void _setSubscriptionTopics() {
+    List<String> topics = ['hello', 'world'];
```

Comment:
Topics need to be given via parameter

Reply:
Only methods name can be accessible from the `CleverPushButton` that is why I have made a methods without parameter.

Reply:
Sorry, you're right, I missed that it's the example class.